lib/app/endpoint.rb

```diff
@@ -20,6 +20,7 @@ class Endpoint < Sinatra::Base
       Inferno::EXTRAS = settings.include_extras
 
       if settings.logging_enabled
+        $stdout.sync = true # output in Docker is heavily delayed without this
```

Comment:
Should this be independently configurable?  I'm not sure if there is a performance impact with doing this, but we might not always care about stdout anyways.  What do you think?

Reply:
If we're expecting to run Inferno in docker in production (which I think we are), we will want this. I don't think it's worth adding configuration so that we could avoid a potential performance penalty only in development.